decls/i18n.js

```diff
@@ -59,6 +59,8 @@ declare type FormattedNumberPart = {
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/formatToParts#Return_value
 declare type NumberFormatToPartsResult = Array<FormattedNumberPart>;
 
+declare type WarnHtmlInMessageLevel = 'allow' | 'warn' | 'error';
```

Comment:
From [this comment](https://github.com/kazupon/vue-i18n/pull/567#discussion_r277540585), I suggest to change enums like this

`WarnHtmlInMessageLevel = 'off' | 'warn' | 'error';` 

Because `off` for `Warn～` makes sense to disable warning.
`allow` for `Warn～` is very confusing.

Reply:
sorry for my miss-understanding...